apps/api/src/app/support/usecases/fetch-user-organizations/fetch-user-organizations.command.ts

```diff
@@ -0,0 +1,55 @@
+import { BaseCommand } from '@novu/application-generic';
+import { IsArray, IsDefined, IsOptional, IsString } from 'class-validator';
+
+export class PlainCutomer {
```

Comment:
There seem to be some duplication in the DTO's I would suggest simplifying this a bit, or reusing if they are identical, just for simplifying our maintenance.